doc/Language/objects.pod6

```diff
@@ -4,8 +4,8 @@
 
 =SUBTITLE Object Orientation in Perl 6
 
-Perl 6 is an object oriented language at its core, even though it allows you
-to write programs in other programming styles.
+Perl 6 is an object oriented language at its core, although you can write
```

Comment:
I'm thinking we don't want to say that it's primarily or especially an object oriented  language. It's also a functional programming language and a procedural and...

Maybe: 

Perl 6 provides strong support for object oriented programming. Although Perl 6 allows programmers to program in multiple paradigms, object oriented programming is at the heart of the language.